lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1315,6 +1315,9 @@ void PatchEntryPointMutate::addUserDataArgs(SmallVectorImpl<UserDataArg> &userDa
           addUserDataArg(userDataArgs, node.offsetInDwords + dwordOffset, pushConstOffset.dwordSize,
                          &pushConstOffset.entryArgIdx, builder);
         }
+      } else {
+        // Mark push constant for spill for compute library.
+        userDataUsage->pushConstSpill = true;
       }
 
       // Ensure we mark the push constant's part of the spill table as used.
```

Comment:
You can remove the check for `|| isComputeWithCalls()` in the line below (GitHub doesn’t let me comment on that line directly).